### ISSUE TITLE
!rule feature to return items from rules reference

### ISSUE THREAD

**Issue by cielago** [MyrionPhoenixmoon/l5r-discord-bot#11]
preliminary design:
!rules topics: PMs list of topics to user
!rules topic: posts short summary to channel
!rules full topic: pms full text of rule to user, so user can hand paste it in if they think its really necessary

**Comment by MyrionPhoenixmoon** [comment]
Has been implemented as !ruling <card> but could use some improvements:
Notes:
- remove the two lines inside the box, "Miya Herald" and "FAQ and RRG entries" ("Rulings for Bayushi Yunako" is explanatory enough)
- switch the position of ruling and source (it's intended to be ruling-source, as name-ruling it just won't be very informative i think)
- add links: "Bayushi Yunako" should link to the 5rdb card page, and ruling sources should link to whatever they link there
basically use the same template as 5rdb itself

**Comment by MyrionPhoenixmoon** [comment]
Card rulings have been improved and are done.

Links to RRG not properly possible at the moment.
